imports/plugins/core/navigation/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,62 @@
+extend type Query {
+  "Get the navigation items for a shop"
+  navItems(shopId: ID!): NavItem
+}
+
+type NavItem {
+  "The Navigation Item ID"
+  _id: ID!
+
+  "The item's content. Can be plain text or HTML."
+  content: String!
+
+  "The link URL. Can be left empty to have context wrapped in a <span> instead of <a>."
+  url: String
```

Comment:
I'm not sure about this, but it might be good to have an `isUrlRelative` boolean? The client may need to know in some cases and shouldn't have to parse it to check.

Maybe also `isUrlExternal`, so the client could choose to open in a new window, or even `shouldOpenInNewWindow` to let the operator control that?